SafeguardDevOpsService/Logic/AddonManager.cs

```diff
@@ -114,14 +114,17 @@ private bool LoadAddonService(Addon addon)
 
         public void AddonPropertyChangedHandler(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
-            _logger.Information($"Addon accounts have changed.  Saving changes.");
-
-            _configDb.SaveAddon(sender as Addon);
+            if (_devOpsAddon.AddOn.CredentialsUpdated)
+            {
+                _devOpsAddon.AddOn.CredentialsUpdated = false;
 
-            _devOpsAddon.AddOn.PropertyChanged -= AddonPropertyChangedHandler;
-            _devOpsAddon.AddOn.IsDirty = false;
-            _devOpsAddon.AddOn.PropertyChanged += AddonPropertyChangedHandler;
+                _logger.Information($"Addon accounts have changed.  Saving changes.");
+                _configDb.SaveAddon(sender as Addon);
 
+                // _devOpsAddon.AddOn.PropertyChanged -= AddonPropertyChangedHandler;
```

Comment:
Do you still need this?

Reply:
I probably won't need it but I left it there on purpose until I can get through some more of the implementation.